### DISCUSSION TITLE
Dashboard option for always autofitpanels

### DISCUSSION THREAD

**Discussion by Caffe1neAdd1ct** [grafana/grafana#37027] [Dashboards, Panels & Visualizations]
We're facing an issue where users have different size screens and they are constantly having to scroll the main dashboard window then scroll inside some of our larger table panels.

Would be great to have a UI or JSON options to force specific dashboard to always use the &autofitpanels url option.

**Comment by Caffe1neAdd1ct** [comment]
https://github.com/grafana/grafana/pull/12796 - feedback related to pull

**Comment by markchagers** [comment]
We're having this issue too: we embed a grafana dashboard in a large web application. The user can switch dashboards with the built-in links. However when switching to a different dashboard the &autofitpanels url parameter isn't carried over. As a result only the initial dashboard we link to has the &autofitpanels parameter (because we add it to the url) but subsequent dashboards that the user navigates to miss it, ruining the layout.
We *really* need an autofit option as a dashboard setting, so that the panels always autofit for a dashboard with that option.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
